src/ZfcUser/Authentication/Adapter/Db.php

```diff
@@ -34,6 +34,11 @@ class Db extends AbstractAdapter implements ServiceManagerAwareInterface
      */
     protected $options;
 
+    public function __construct(ServiceManager $serviceManager)
```

Comment:
or can leave in constructor, but still use `$this->getServiceLocator()` and leave `ServiceManagerAwareInterface`